### ISSUE TITLE
Yesterday I was able to run this image but not today!

### ISSUE THREAD

**Issue by kanashin** [wnameless/docker-oracle-xe-11g#43]
First of all, thank you for your job!

I created a virtual machine with Ubuntu 16.04 x64, I installed docker and then I ran:

	docker pull wnameless/oracle-xe-11g
	docker run -d -p 49160:22 -p 49161:1521 -e ORACLE_ALLOW_REMOTE=true wnameless/oracle-xe-11g

I could do `sqlplus system/oracle` without any problem.

Today I tried the same in another fresh virtual machine and I did the same, but every time I do `sqlplus system/oracle` I see:

	SQL*Plus: Release 11.2.0.2.0 Production on Fri Dec 2 19:57:58 2016

	Copyright (c) 1982, 2011, Oracle.  All rights reserved.

	ERROR:
	ORA-01033: ORACLE initialization or shutdown in progress
	Process ID: 0
	Session ID: 0 Serial number: 0

What I'm missing?

**Comment by kanashin** [comment]
If after a while I do a `shutdown immediate` and then `startup` I can see `ORA-00205: error in identifying control file, check alert log for more info`

**Comment by wleroux** [comment]
I noticed this too. I've created a pull request to fix this issue (https://github.com/wnameless/docker-oracle-xe-11g/pull/44)

A recent change has added support for volumes. To support volumes, the oradata data has moved to oradata.orig. In copying the contents of oradata.orig back to oradata, the file attributes get lost. The oracle-xe service does not have permission to access the copied oradata (causing permission issues.)
